src/include/platform/CHIPDeviceConfig.h

```diff
@@ -955,6 +955,16 @@
  *   The value is base-64 encoded string.
  */
 #ifndef CHIP_DEVICE_CONFIG_USE_TEST_SPAKE2P_VERIFIER
+
+#if CHIP_DEVICE_CONFIG_USE_TEST_SETUP_PIN_CODE != 20202021
+#error "Non-default Spake2+ passcode configured but verifier left unchanged"
+#endif
+
+#if CHIP_DEVICE_CONFIG_USE_TEST_SPAKE2P_ITERATION_COUNT != 1000
+#error "Non-default Spake2+ iteration count configured but verifier left unchanged"
+#endif
+
```

Comment:
Need to add similar check for CHIP_DEVICE_CONFIG_USE_TEST_SPAKE2P_SALT

Reply:
Right, but you can't compare strings in a preprocessor. I can add a constexpr function + static assert though.